### ISSUE TITLE
Show author / channel name for tracks

### ISSUE THREAD

**Issue by napstr** [Frederikam/FredBoat#401]
Picked this up from #suggestions

Let's think about displaying the author / channel of a track.
We do have access to this information through AudioTrack#getInfo#author, which seems to return the channel name for Youtube and Soundcloud tracks.
Current state:
![image](https://user-images.githubusercontent.com/6048348/32957373-d12369f4-cbbb-11e7-86a4-9261175a1de3.png)
Possible future state:
![image](https://user-images.githubusercontent.com/6048348/32957384-daafd12e-cbbb-11e7-8dba-b3b4c2cb0441.png)

Opinions? Ideas how to display this decently?

I think the biggest might be that a channel name a lot of the time isnt even close the the author of a track, so we should probably make it clear that it's a channel name.

**Comment by druellan** [comment]
Hi Psy here.

Seems that SoundCloud does NOT provide the artists name on a separate field.
From here https://developers.soundcloud.com/docs/api/reference#tracks seems that you can get the channel name and the song name, not the artists. You can also get the record label name, but this is not relevant on this case.
So, SC and YT are both providing the same information: songname and channelname

As a template, I like
[songname] on [channelname] channel ([duration]). Ex: "2 Kids on SwimmingGirls channel (03:16)"

**Comment by shikhir-arora** [comment]
I agree with the above. SoundCloud doesn't pass the artist name, even though there is an option for it when you upload tracks. No one really uses that option anyway because it's tucked away in the metadata section when you upload a track. 

Anyways, as @druellan mentioned, that is correct. SoundCloud data is pretty universal (any link can be resolved with their `/resolve` endpoint) and the user ("channel") name is part of the user-JSON field contained in every track/playlist result from the API. I think the template suggested above is pretty good. You can even keep it fairly simple/elegant and I think most users will get it IMO (similar to how Facebook got rid of the send button on their chat and users know to push enter 😄 
```ml
Song • Channel • [duration]

2 Kids • SwimmingGirlsVEVO • [3:16]

Song - Channel - [duration]

2 Kids - SwimmingGirlsVEVO - [3:16]

```
This is how Spotify does it as well, they just provide the artist name/album since they have it: 

<img width="934" alt="screen shot 2017-11-20 at 9 08 21 am" src="https://user-images.githubusercontent.com/8797918/33022224-6fb03756-cdd2-11e7-9812-c4b9e321a19c.png">

**Comment by Frederikam** [comment]
As Napster said the artist is provided by lavalink already
![image](https://user-images.githubusercontent.com/2582617/33032212-d466f70e-ce20-11e7-8143-cb18f21856ac.png)
